src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1006,19 +1001,16 @@ private function getValidatorMappingFiles(ContainerBuilder $container, array &$f
         foreach ($container->getParameter('kernel.bundles_metadata') as $bundle) {
             $dirname = $bundle['path'];
 
-            if (is_file($file = $dirname.'/Resources/config/validation.yml')) {
+            if ($container->fileExists($file = $dirname.'/Resources/config/validation.yml', false)) {
```

Comment:
Maybe we should open a PR for older branches to make use of the `FileExistenceResource` there.